src/ButtonLink/README.md

```diff
@@ -59,3 +62,13 @@ const YourComponent = props => <div {...props}>YourComponent</div>
 <ButtonLink component={YourComponent}>Title</ButtonLink>
 ```
 This example will render ButtonLink in div with children **YourComponent** and not **Title**.
+
+
+## Accessibility
+if you use `ButtonLink` without `href` so it's rendered as a `<Button>` element then you can use props below. In case you are passing `href` and element is rendered as `<a>` then it should be treated as a link and `ariaControls`, `ariaExpanded` shouldn't be useful. 
```

Comment:
I would use `<button>` so it's clear that you mean HTML element and not a component

Reply:
Also, use a capital `I` at the start of the sentence